include/GoogleSync/GoogleSync.php

```diff
@@ -923,7 +1078,7 @@ public function updateGoogleCalendarEvent($event_local, $event_remote)
      *
      * @return \Google_Service_Calendar_Event Google_Service_Calendar_Event Object
      */
-    public function createGoogleCalendarEvent($event_local)
+    public function createGoogleCalendarEvent(Meeting $event_local)
```

Comment:
line https://github.com/salesagility/SuiteCRM/pull/6539/files#diff-034e880ea433e2cc2571088f7cbc6620R1094: `unset($event_remote_empty); // We're done with this` is unnecessary

Reply:
Fixed.